.semaphore/semaphore.yml

```diff
@@ -33,3 +33,11 @@ blocks:
           commands:
             - go test ./...
             - make build
+            - "curl -H \"Authorization: Bearer $SEMAPHORE_OIDC_TOKEN\" https://terraform-registry.stage.dp.confluent.io/.well-known/terraform.json"
```

Comment:
We should not expose the dns name to public here

Reply:
yeah I'll test it in a private repo next time. just removed the test folder and updated semaphore.yaml. It's still in the commit history tho. Should we update the dns name?

Reply:
We are going to change the name later anyway, so this is fine for now.